app/modules/sidebars/components/sidebars/Sidebar.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import type { Identifier } from 'types/model';
+import type { State } from 'types/state';
+import topics from 'modules/topics';
+import ObjectNotFoundError from 'errors/usage-errors/ObjectNotFoundError';
+import { sidebarMapping } from '../../model/sidebarName';
+import type { Sidebar as SidebarType } from '../../model';
+
+type Topic = topics.model.Topic;
+const { getById } = topics.selectors;
+
+type PassedProps = {
+  sidebarName: SidebarType,
+  topicId: Identifier,
+};
+
+type StateProps = {
+  topic: Topic,
+};
+
+type Props = PassedProps & StateProps;
+
+const mapStateToProps = (state: State, props: PassedProps): StateProps => {
+  const { topicId } = props;
+  const topic = getById(state, { id: topicId });
```

Comment:
This is more of a general remark because I've noticed this in multiple places, but it would be much more readable if you would either use `topics.selectors.getById(...)` or define your `getById` const as `getTopicById` instead. Preferably the former, since that way it involves only one line of code :P